vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,119 @@
+upstream usage of the package's header files. For example, a `contoso` library that provides
+`contoso/contoso.h` should create the necessary subdirectories to reflect the relative path, in this
+case, the header file should be placed in `include/contoso/contoso.h`.
+
+vcpkg forbids installing some reserved header file names in the root of the `include` directory, for
+example: `err.h`, `user.h`, `time.h`, and others. Libraries that provide a forbidden header file name should
+place their header files inside a `include/<port>` directory. If the library intends to replace a
+system header file, it should set the `VCPKG_POLICY_ALLOW_RESTRICTED_HEADERS` policy in its
+`portfile.cmake`.
+
+### <a name="layout-lib"></a> `lib` and `debug/lib` directories
+
+Contains static libraries, import libraries (on Windows), and shared libraries (on non-Windows).
+configuration-dependent directories.
+
+### <a name="layout-manual-link"></a> `lib/manual-link` and `debug/lib/manual-link` directories
+
+Contains object files that must be manually linked, these directories are configuration dependent.
+Each port must make sure to place manually-linkable files in these directories explicitly.
+
+Files that may cause issues when linked automatically must be placed in the `lib/manual-link`
+folders instead of the `lib` directory. For example, if a library is intended to define the
+`main()` function for a program. 
+
+### <a name="layout-pkgconfig"></a> `lib/pkgconfig` and `share/pkgconfig` directories
+
+Contains pkgconfig integration files (`.pc`).
+
+### <a name="layout-share"></a> `share/<port>`
+
+Contains miscellaneous files installed by each port, at the very least, each port is required to
+provide a `LICENSE` file in `share/<port>` (replace `<port>` with the port name). Some examples of
+files contained in this directory are: CMake integration files,
+[usage](../maintainers/handling-usage-files.md) files, SPDX files, etc.
+
+### <a name="layout-copyright"></a> `share/<port>/copyright`
+
+vcpkg expects ports to provide a `copyright` file that contains the license information for the
+installed package. See the [maintainer
+guide](../contributing/maintainer-guide.md#install-copyright-file) for more information.
+
+### <a name="layout-cmake"></a> `share/<package>/<package>Config.cmake`, `share/<package>/<package>-config.cmake`
+
+CMake integration files should be placed in the `share` folder and respect CMake's rules for
+[`find_package(package)`](<https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure>)
+in `CONFIG` mode.
+
+### <a name="layout-tools"></a> `tools/<port>`
+
+Contains executable tools produced by a port, it is not required, but is highly recommended that
+each installed executable goes into a subdirectory matching the name of the port that produced it.
+For example, a `contoso` port might install a `ContosoGenerator.exe` to
+`tools/contoso/ContosoGenerator.exe`.
```

Comment:
Autotools-based ports install tools to `tools/<port>/bin` and `tools/<port>/debug/bin`, https://github.com/microsoft/vcpkg/issues/17607

And I just stumbled over inconsistent handling of "private" tools which ports normally install to `libexec` on non-windows platforms. (I would map this to `manual-tools` in vcpkg.)